app/assets/javascripts/osem-schedule.js

```diff
@@ -117,6 +119,40 @@ $(document).ready( function() {
   });
 });
 
+
+// PUBLIC SCHEDULE
+
+function starClicked(e){
+  // stops the click from propagating
+  if (!e) var e = window.event;
+  e.preventDefault();
+  e.cancelBubble = true;
+  if (e.stopPropagation) e.stopPropagation();
+
+  var callback = function(data) {
+    if($(e.target).hasClass('fa-star-o')){
+      $(e.target).removeClass('fa-star-o');
+      $(e.target).addClass('fa-star');
+    }
+    else{
+    $(e.target).removeClass('fa-star');
+      $(e.target).addClass('fa-star-o');
+    }
+  }
+
+  var params = { public_user_id: $(e.target).data('user') };
+  if($(e.target).hasClass('fa-star-o')){
```

Comment:
but that is to know if I have to add or delete the user in the controller. But I could also do that without this param, just deleting the user if already exists or adding it otherwise.